### PR TITLE
Add a section on Git hooks

### PR DESCRIPTION
**Pull request by yeskunall**
/cc @vpanjganj

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ If you want to share a best practice, or think one of these guidelines  should b
 - [Testing](#testing)
 - [Structure and Naming](#structure-and-naming)
 - [Code style](#code-style)
+    - [Git Hooks](#git-hooks)
```
Comment (vpanjganj) [inline README.md]: Let's have 2 main sections here only:
Some Code Style guidelines
Enforcing Code Style

**Comment by yeskunall** [comment]
Thanks for the feedback! I'll start by reformatting it to the current flow
and try to make it less wordy in the process. I'll hit you up once I've
made these changes.

>
> --

Kunall Banerjee

**Comment by yeskunall** [comment]
@vpanjganj I haven't forgotten about this. Expect a PR sometime today! ❤️

**Comment by vpanjganj** [comment]
@yeskunall no problem. I keep this open.

**Comment by vpanjganj** [comment]
I think You still have bits left, right?

**Comment by yeskunall** [comment]
Yup, I still need to add more info. Yes, don't worry, the links will be updated when I make the final push. Thanks for being patient @vpanjganj.

**Comment by yeskunall** [comment]
@vpanjganj so I made the requested changes. Let me know if you think if anything needs to be rewritten.

**Pull request merged**